spring-cloud-gcp-samples/spring-cloud-gcp-pubsub-stream-dead-letter-sample/src/main/resources/application.properties

```diff
@@ -16,8 +16,6 @@ spring.cloud.stream.gcp.pubsub.bindings.logUserMessage-in-0.consumer.ackMode=MAN
 # The application also subscribes to the dead letter topic to show that it's just another normal topic.
 spring.cloud.stream.bindings.deadLetterMessages-in-0.destination=my-dead-letter-topic
 
-spring.cloud.stream.gcp.pubsub.default.consumer.auto-create-resources=true
```

Comment:
This is still in the docs though. Please update documentation as well.

Reply:
The docs are updated. 

I'm not sure whether we still need the polling example and I'll do more research about it.

Reply:
The legacy imperative programming model has been removed, so we don't need the polling example now.